### DISCUSSION TITLE
Welcome to Tuxemon Discussions!

### DISCUSSION THREAD

**Discussion by ShadowApex** [Tuxemon/Tuxemon#2213] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Sanglorian** [comment]
Thanks heaps for setting this up, @ShadowApex. I think this will be a good complement to the Discord, for longer-term or niche discussions.

**Comment by Davidliang11292** [comment]
Wow, this discussion page is really great for me, thanks for your work!

**Comment by HippasusTwo** [comment]
Thanks.
